src/evm-block-extractor/src/task/block_extractor.rs

```diff
@@ -171,6 +173,25 @@ impl BlockExtractor {
         Ok((from_block_inclusive, to_block_inclusive))
     }
 
+    /// Collects last certified block
+    async fn collect_last_certified_block(&self) -> anyhow::Result<()> {
```

Comment:
this is not tested. 
You have to add a check inside the `src/evm-block-extractor/tests/tests/block_extractor_it.rs` to verify that the last certified block is fetched by the extractor

Reply:
Test was added